tests/plugins/editorplaceholder/editorplaceholder.js

```diff
@@ -205,5 +205,20 @@
 		} );
 	};
 
+	// (#4253)
+	tests[ 'test placeholder loads correctly in full-page editor with bbcode plugin'] = function() {
+		bender.editorBot.create( {
+			name: 'bbcodeFailTest',
+			config: {
+				extraPlugins: 'bbcode',
+				editorplaceholder: 'any',
+				fullPage: true
+			}
+		}, function( bot ) {
+			//no errors during editor load - so test can pass
+			assert.isTrue( true );
+		} );
+	};
+
```

Comment:
I'm also missing second test covering case mentioned in https://github.com/ckeditor/ckeditor4/issues/4253#issuecomment-689774158. Please check [`toDataFormat` event](https://ckeditor.com/docs/ckeditor4/latest/api/CKEDITOR_editor.html#event-toDataFormat) to see how to manipulate editor data on output.

Reply:
I add a test against content without body, but it appears `bbcode` plugin filter-out all non-body content.

So it looks like `bbcode` plugin with `config.fullPage: true;` has no sense.

Reply:
The case from https://github.com/ckeditor/ckeditor4/issues/4253#issuecomment-689774158 doesn't involve `bbcode` plugin, but the effect is the same as you mentioned - you get HTML without body when `fullpage` plugin is used. So it looks it even has more sense than `fullpage` + `bbcode` plugin :thinking:

Reply:
See https://github.com/ckeditor/ckeditor4/pull/4370#discussion_r524966468, it seems we will need more tests anyway... :smile: